lib/banyan_api/anonymizers/shopify_order.ex

```diff
@@ -0,0 +1,21 @@
+defmodule BanyanAPI.Anonymizers.ShopifyOrder do
+  alias BanyanAPI.Anonymizers.ShopifyAddress
+
+  @order_fields_omit [
+    "contact_email",
+    "customer",
+    "email",
+    "fulfillments"
+  ]
+
+  @spec clean(any()) :: map()
+  def clean(order) when is_map(order) do
+    order
+    |> Map.put("customer_id", get_in(order, ["customer", "id"]))
```

Comment:
Have we considered keep the customer object there, but just leave it as having one property, ID? That keeps the shapes of each object matching Shopify -- might be more future-proof/consistent?